PROCESSES.md

```diff
@@ -2,10 +2,100 @@
+This allows them to manage issues, review PRs etc.
+
+Admin permissions allow you to modify repository settings, that's nothing which is needed on a daily basis.
+The goals is to limit the number of admins to avoid misconfigurations.
+At the same time it makes sense to have more than one admin so that changes from one admin can be reviewed by another one.
+At the moment there are three admins.
+
+### Branch Protection Rules
+
+The `main` branch is protected and the following settings are configured:
+
+- Require pull request reviews before merging: 1
+  > When enabled, all commits must be made to a non-protected branch and submitted via a pull request with the required number of approving reviews and no changes requested before it can be merged into a branch that matches this rule.
+
+  As many people rely on charts hosted in this repository each PR must be reviewed before it can be merged.
+  
+  - Dismiss stale pull request approvals when new commits are pushed
+
+    > New reviewable commits pushed to a matching branch will dismiss pull request review approvals.
+
+    This prevents that changes can be made unnoticed to already approved PRs.
+    As a consequence of this every change made to an already approved PR will need another approval.
+
+  - Require review from Code Owners
+
+    > Require an approved review in pull requests including files with a designated code owner.
+
+    This repository hosts multiple helm charts with different maintainers.
+    This setting helps us to ensure, that every change to a chart needs to be approved by at least one of the maintainers of a that chart.
+
+    As a consequence CODEOWNERS and maintainers of a chart defined in `Chart.yaml` needs to be in sync.
+
+- Require status checks to pass before merging
+  > Choose which [status checks](https://docs.github.com/en/free-pro-team@latest/rest/reference/repos#statuses) must pass before branches can be merged into a branch that matches this rule. When enabled, commits must first be pushed to another branch, then merged or pushed directly to a branch that matches this rule after status checks have passed.
+  - DCO
+
+    [Developer Certificate of Origin](https://developercertificate.org/) (DCO) check is performed by [DCO GitHub App](https://github.com/apps/dco)
+
+  - Lint Code Base
+
+    Linting is done using [Super-Linter](https://github.com/github/super-linter).
+    It is configured in [linter.yaml](.github/workflows/linter.yml)
+
+  - lint-test
+
+    Helm charts are tested using [Chart Testing](https://github.com/helm/chart-testing), which is configured in [lint-test.yaml](.github/workflows/lint-test.yaml).
+
+- Include administrators
+  > Enforce all configured restrictions above for administrators.
+
+  To play fair all the settings above are also applied for administrators.
+
+- Force pushes and deletions are disabled
+
+  Force pushes and deletions on the `main` branch branch should never be done.
```

Comment:
**Quick note:** reminder to both of us that the `gh-pages` branch is not currently protected against force-pushing, and to double-check up to see if this is still necessary for `helm/chart-releaser` (it used to require force-pushing, but that may have changed and it didn't register with me).

Reply:
It's protected for https://github.com/jenkinsci/helm-charts and works well. I can check the settings over there and and document them as another PR. You can verify them and after that we can configure those.

Looks like following the process outlined in this document ;-)